app/src/main/java/com/teamagam/gimelgimel/app/map/esri/plugins/UIUpdatePoller.java

```diff
@@ -0,0 +1,44 @@
+package com.teamagam.gimelgimel.app.map.esri.plugins;
+
+
+import com.teamagam.gimelgimel.app.common.utils.Constants;
+import com.teamagam.gimelgimel.domain.base.subscribers.SimpleSubscriber;
+
+import java.util.concurrent.TimeUnit;
+
+import rx.Observable;
+import rx.Scheduler;
+import rx.Subscription;
+
+abstract class UIUpdatePoller {
+
+    private final Scheduler mWorkScheduler;
+    private Subscription mSubscription;
+    private boolean mIsRunning;
+
+    public UIUpdatePoller(Scheduler workScheduler) {
+        mWorkScheduler = workScheduler;
```

Comment:
Why let the extending classes pass this scheduler? use Schedulers.computation() in here and save them this decision... They both use it.